src/core/Notification/BaseNotification/BaseNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import { HtmlDivWithRefProps } from '../../../reset';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  inline: `${baseClassName}--inline`,
```

Comment:
Hmm.. I think that class names that are not mutual for both versions are no supposed to be in this same `notificationClassNames` object.

E.g if only some are for inline version, they should be introduced only there.

What others will think about this approach? _(If accepted, Alert also need some slight changes regarding this)_

Reply:
Fixed, now we have only one component.